### DISCUSSION TITLE
open source agreement issue

### DISCUSSION THREAD

**Discussion by amir-asdp** [syhyz1990/baiduyun#356] [Q&A]
Hi
When I click on **"Download Assistant (or 下载助手)" button**, Baidu shows me a message like the images below that says : 
**"Please enter the open source agreement of this assistant"**

<br>

<img src="https://github.com/syhyz1990/baiduyun/assets/62106560/ae0ff43b-57af-4e28-aa4a-ace7d869ea39" height="330" >
<img src="https://github.com/syhyz1990/baiduyun/assets/62106560/95e7ef6f-4e8b-4a66-84d2-9459ae74816f" height="330" >

<br><br>

I tried to enter `AGPL`, `AGPL-3.0`, `AGPLv3.0` and `GNU` but none of them worked. 
Please help me to solve this issue.

<br><br>

**Comment by SSrFreak** [comment]
Dude u need to register a WeChat account to scan this QR code to get right passcode.
Anyway,try "AGPL3" should solve the problem.
